static/themes/oled-dark.css

```diff
@@ -0,0 +1,457 @@
+/* Variables */
```

Comment:
I welcome contributions from theme developers to help improve the organization and readability of our theme's CSS code. Your expertise and insights would be greatly appreciated.